docs/sources/tempo/operations/user-configurable-overrides.md

```diff
@@ -0,0 +1,163 @@
+  [collection_interval: <duration>]
+  [disable_collection: <bool> | default = false]
+
+  processor:
+
+    service_graphs:
+      [histogram_buckets: <list of float>]
+      [dimensions: <list of string>]
+      [peer_attributes: <list of string>]
+      [enable_client_server_prefix: <bool>]
+
+    span_metrics:
+      [histogram_buckets: <list of float>]
+      [dimensions: <list of string>]
+      [intrinsic_dimensions: <map string to bool>]
+      [filter_policies: [
+        [
+          include/exclude:
+            match_type: <string> # options: strict, regexp
+            attributes:
+              - key: <string>
+                value: <any>
+        ]
+      ]
+      [enable_target_info: <bool>]
+      [target_info_excluded_dimensions: <list of string>]
+```
+
+### API
+
+All API requests are handled on the `/api/overrides` endpoint. The module supports `GET`, `POST`, `PATCH`, and `DELETE` requests.
+
+This endpoint is tenant-specific. If Tempo is run in multitenant mode, all requests should have an appropriate `X-Scope-OrgID` header.
+
+If the tenant is run in distributed mode, only the query-frontend will accept API requests.
+
+#### Operations
+
+```
+GET /api/overrides
+```
+
+Returns the current overrides and it's version.
+
+Query-parameters:
+- `scope`: whether to return overrides from the API only `api` or merge it with the runtime overrides `merged`. Defaults to `api`.
+
+Example: `curl -X GET -v -H "X-Scope-OrgID: 3" http://localhost:3100/tempo/api/overrides\?scope=merged`
+
+```
+POST /api/overrides
+```
+
+Update the overrides with the given payload. Note this will overwrite any existing overrides.
+
+Example: `curl -X POST -v -H "X-Scope-OrgID: 3" -H "If-Match: 1697726795401423" http://localhost:3100/api/overrides --data "{}"
+
+```
+PATCH /api/overrides
+```
+
+Update the existing overrides by patching it with the payload. It follows the JSON merge patch protocol ([RFC 7386](https://datatracker.ietf.org/doc/html/rfc7386)).
+
+Example: `curl -X PUT -v -H "X-Scope-OrgID: 3" -H "If-Match: 1697726795401423" http://localhost:3100/api/overrides --data "{\"forwarders\":null}"
+
+```
+DELETE /api/overrides
+```
+
+Delete the existing overrides.
+
+Example: `curl -X DELETE -H "X-Scope-OrgID: 3" -H "If-Match: 1697726795401423" http://localhost:3100/api/overrides`
+
+#### Versioning
+
+To handle concurrent read and write operations, overrides are stored with a version in the backend.
+Whenever the overrides are returned, the response will have an Etag header with the current version.
+
+```
+$ curl -v http://localhost:3100/api/overrides
+...
+< HTTP/1.1 200 OK
+< Content-Type: application/json
+< Etag: 1697726795401423
+< Date: Wed, 07 Feb 2024 17:49:04 GMT
+< Content-Length: 118
+...
+```
+
+Requests that modify or delete overrides need to pass the current version using the `If-Match` header:
+
+```
+$ curl -X POST -H "If-Match: 1697726795401423" http://localhost:3100/api/overrides --data "..."
+```
+
```

Comment:
Something like this?  The suggestion isn't in ` ``` ` because the editor messes up when I use that. 

```suggestion
This example uses uses overrides in the `overrides.json` file with the location in `pwd`: 

`curl -X POST -H "X-Scope-OrgID: 3" -H "If-Match: 1697726795401423" http://localhost:3100/api/overrides --data @overrides.json`

```

Reply:
Yep this looks good :+1: